test/kata2.booleanToWord.test.js

```diff
@@ -1,5 +1,10 @@
 const { booleanToWord } = require("../src");
 
 describe("booleanToWord", () => {
-  // how do we create specs again???
+  it("returns Yes when passed true", () => {
+    expect(booleanToWord(true)).toEqual("Yes");
+  });
+  it("returns No when passed false", () => {
```

Comment:
Its a common convention to leave a clear line between each of your `it` blocks to make the test file easier to read:
```
  });

  it("returns No when passed false", () => {
```